.github/workflows/deploy.yml

```diff
@@ -13,20 +13,21 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest]
+        os: [macos-11, ubuntu-20.04]
```

Comment:
These are the earliest available. We'll have to mention somewhere that we only support above these versions.